server/odc-service/src/main/java/com/oceanbase/odc/service/flow/FlowInstanceService.java

```diff
@@ -732,6 +732,10 @@ private FlowInstanceDetailResp buildWithoutApprovalNode(CreateFlowInstanceReq fl
             ConnectionConfig connectionConfig) {
         log.info("Start creating flow instance, flowInstanceReq={}", flowInstanceReq);
         TaskType taskType = flowInstanceReq.getTaskType();
+        if (taskType == TaskType.ASYNC) {
```

Comment:
it also affect database change task generated by sql plan task ?

Reply:
Yes! All task generated by this method has no PRE_CHECK task, so need add this `if` block.